sycl/doc/extensions/experimental/sycl_ext_intel_event_mode.asciidoc

```diff
@@ -72,6 +73,9 @@ only wake up occasionally to check if the event has finished. This reduces the
 time the CPU spends checking finish condition of the wait, at the cost of
 latency.
 
+This extension exists to solve a specific problem, and a general solution is
+still being evaluated.  It is not recommended for general usage.
```

Comment:
@gmlueck & @jbrodman - Do you agree with adding this statement or is it general-purpose enough to leave it out?

Reply:
As discussed offline, we don't need this statement.